RecoParticleFlow/Configuration/python/RecoParticleFlow_EventContent_cff.py

```diff
@@ -156,7 +156,7 @@ def _modifyPFEventContentForHGCalFEVT( obj ):
         'keep recoPFClusters_particleFlowClusterHGCalFromMultiCl__*',
         'keep *_particleFlowSuperClusterHGCalFromMultiCl_*_*',
         'keep *_simPFProducer_*_*',
-        'keep *_particleFlowTmpBarrel_*_*',
+        'keep *_particleFlowTmp_*_*'
```

Comment:
```python
process.particleFlowTmp = cms.EDProducer("PFCandidateListMerger",
    src = cms.VInputTag("particleFlowTmpBarrel", "simPFProducer")
)
```
Is the content of   ```_particleFlow_``` inappropriate (it is in the standard event content already)?
It's made out of ```particleFlowTmp```. So, this would just add a copy of the particleFlow collection with pre-linking step information about objects and links.

Reply:
@rovere 
please clarify on this.
It appears to me that saving the Tmp for the merged pf collection is not needed because the relinked collection contains all of its information

Reply:
@rovere 
I'm waiting for feedback on this. (what's missing in particleFlow to require also saving particleFlowTmp?)
Please comment.

Reply:
@slava77 most likely, after the 2 additional commits that stores the electrons directly into pfCandidates, the particleFlorTmp could be dropped completely.